osi_object.proto

```diff
@@ -745,66 +748,133 @@ message MovingObject
             //
             // Definition: Hatchback car with maximum length 4 m.
             //
+            // This is one of the OSI types which map from "car" in both OpenScenario 1.x and 2.x.
+            //
             TYPE_SMALL_CAR = 2;
 
             // Vehicle is a compact car.
             //
             // Definition: Hatchback car with length between 4 and 4.5 m.
             //
+            // This is one of the OSI types which map from "car" in both OpenScenario 1.x and 2.x.
+            //
             TYPE_COMPACT_CAR = 3;
 
             // Vehicle is a medium car.
             //
             // Definition: Hatchback or sedan with lenght between 4.5 and 5 m.
             //
+            // This is one of the OSI types which map from "car" in both OpenScenario 1.x and 2.x.
+            //
             TYPE_MEDIUM_CAR = 4;
 
             // Vehicle is a luxury  car.
             //
             // Definition: Sedan or coupe that is longer then 5 m.
             //
+            // This is one of the OSI types which map from "car" in both OpenScenario 1.x and 2.x.
+            //
             TYPE_LUXURY_CAR = 5;
 
             // Vehicle is a delivery van.
             //
             // Definition: A delivery van.
             //
+            // This corresponds to "van" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "truck" in OpenScenario 2.x.
+            //
             TYPE_DELIVERY_VAN = 6;
 
-            // Vehicle is a heavy truck.
+            // Vehicle is a (heavy) truck.
+            //
+            // Definition: A rigid-body truck that cannot pull a semi-trailer.
```

Comment:
Suggestion:
"Definition: A rigid-body truck."
I would leave out the addition regarding the semi-trailer since this is also valid for each other type, except the semi-tractor.

Reply:
Yep, makes sense, done.